Makefile

```diff
@@ -26,7 +26,7 @@ $(GLEAN_PYENV)/bin/python3:
 	python3 -m venv $(GLEAN_PYENV)
 	$(GLEAN_PYENV)/bin/pip install --upgrade pip
 	$(GLEAN_PYENV)/bin/pip install --use-feature=2020-resolver -r glean-core/python/requirements_dev.txt
-	sh -c "if [ \"$(GLEAN_PYDEPS)\" == \"min\" ]; then \
+	sh -c "if [ \"$(GLEAN_PYDEPS)\" = \"min\" ]; then \
```

Comment:
What's this about?

Reply:
Side fix I stumbled upon when I saw the test fail.
`sh` is a POSIX shell, so it uses `[` (aka `test`) in its single-bracket form (because ... posix), and in posix-`test` one uses a single equal sign (`=`) for comparison (whereas `==` is a bash thing and not posix compatible)

Source: https://pubs.opengroup.org/onlinepubs/9699919799/utilities/test.html, why I know this? Good you asked! I once maintained scripts to install Redis _everywhere_ and thus the only thing I could ever rely on was POSIX and not bash. Trust me, you don't want to maintain that.